docs/docs/plugins-themes-and-starters.md

```diff
@@ -62,15 +68,15 @@ When it comes to maintaining a Gatsby site, plugins and themes offer a distinct
 
 #### Versioning
 
-Starters can still be versioned inside a repository so that you can track issues or bugs associated with specific developments in the starter but it's more difficult to make updates with new changes.
+Starters can still be versioned inside of a repository so that you can track issues or bugs associated with specific developments, but they aren't formally released in publishes marked by versions like a plugin or theme is.
```

Comment:
```suggestion
Starters can still be versioned inside of a repository so that you can track issues or bugs associated with specific updates, but they aren't formally released and published, so they aren't versioned like a plugin or theme is.
```